### ISSUE TITLE
Enhancement: Support for 7tv's Event API

### ISSUE THREAD

**Issue by JunkiEDM** [flex3r/DankChat#429]
[7tv's Event API](https://github.com/SevenTV/EventAPI) allows for 7tv emote additions, removals, and aliases to update live without the need to refresh emotes, this is relatively new, but an example of it in action is in [SevenTV/chatterino7](https://github.com/SevenTV/chatterino7), where it displays a grayed-out message indicating that an emote has been added/removed/renamed.

**Comment by flex3r** [comment]
I really like the emote live updates in chatterino but there is a key difference in implementing it for a desktop app and implementing it in a mobile app.
This would introduce another active websocket connection that would consume a ton of resources. These long-running active connections are actually one  of the biggest battery drainers and some OEMs go even so far that they will randomly kill these connections without properly notifying the app. An alternative would be to only connect to the event api if the app is in the foreground but then some emote changes might be missed while the user doesn't have DankChat open.

TLDR I like the real-time aspect but I'm a bit hesitant to at least have this enabled by default. If we implement this it would be opt-in only.

**Comment by francorz** [comment]
could be an optional feature, so if you have a low gamma phone, you just turn it off :D
